.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,50 @@
+---
+name: Bug Report
+about: If things aren't working as expected.
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+## Bug Report
+
+<!-- 
+Note: Make sure to first check the prerequisites that can be found in the main README file!
+
+Thanks for filing an issue! Before hitting the button, please answer these questions.
+Fill in as much of the template below as you can. If you leave out information, we can't help you as well.
+-->
+
+**What did you do?**
+A clear and concise description of the steps you took (or insert a code snippet).
+
+**What did you expect to see?**
+A clear and concise description of what you expected to happen (or insert a code snippet).
+
+**What did you see instead? Under which circumstances?**
+A clear and concise description of what you expected to happen (or insert a code snippet).
+
+
+**Environment**
+* operator-lifecycle-manager version:
+
+<!--- Insert operator-lifecycle-manager release or Git SHA here. -->
+
+* go version:
+
+<!--- Insert the output of `go version` here -->
+
+* Kubernetes version information:
+
+<!--- Insert the output of `kubectl version` here -->
+
+* Kubernetes cluster kind: 
+
+* Are you writing your operator in ansible, helm, or go?
```

Comment:
Not sure this is relevant to all OLM users?

Reply:
👍